sofa-ark-parent/core-impl/container/src/main/java/com/alipay/sofa/ark/container/service/classloader/ClassloaderServiceImpl.java

```diff
@@ -44,10 +48,13 @@
 public class ClassloaderServiceImpl implements ClassloaderService {
 
     private static final ArkLogger LOGGER = ArkLoggerFactory.getDefaultLogger();
+    private static final String JAVA_AGENT_MARK = "-javaagent:";
+    private static final String JAVA_AGENT_OPTION_MARK = "=";
 
```

Comment:
Seems the code formatter plugin stop working... codes are not formatted as they should be.

Reply:
That's too funny, Only this file can NOT formatted correctly.